app/main.py

```diff
@@ -92,38 +97,43 @@
     [
         dcc.Location(id="url", refresh=False),
         html.Div(id="page-content", style={"height": "100%"}),
-        # Interval component to generate call to API every 10 seconds
-        dcc.Interval(
-            id="interval-component-homepage",
-            interval=10 * 1000
-        ),
-        # Hidden div to keep a record of live alerts data
-        dcc.Store(id="store_live_alerts_data", storage_type="memory"),
-        dcc.Store(id="last_displayed_event_id", storage_type="memory"),
-        dcc.Store(id="images_url_current_alert", storage_type="session", data={}),
+
+        # Storage components which contains data relative to alerts
+        dcc.Store(id="store_live_alerts_data", storage_type="session"),
+        dcc.Store(id="last_displayed_event_id", storage_type="session"),
+        dcc.Store(id="images_url_live_alerts", storage_type="session", data={}),
         # Storage component which contains data relative to site devices
         dcc.Store(id="site_devices_data_storage", storage_type="session", data=get_site_devices_data(client=api_client))
     ]
 )
 
+
 # Cache configuration
 cache = Cache(app.server, config={
     'CACHE_TYPE': 'filesystem',
     'CACHE_DIR': '.cache',
     'CACHE_DEFAULT_TIMEOUT': 60
 })
 
-# Fetching reusable alert metadata
-alert_metadata = build_live_alerts_metadata()
-alert_id = alert_metadata["id"]
 
+# End point ping by the API for each alert being recorded. Then broadcasting message to ALL sessions
+@app.server.route("/alert/<message>")
+def broadcast_message(message):
+    socket_pool.broadcast(message)
+    return f"Message {message} broadcast."
 
 # ----------------------------------------------------------------------------------------------------------------------
 # CALLBACKS
 
 # ----------------------------------------------------------------------------------------------------------------------
 # General callbacks
 
+
+# First API relay once an alert is sent to the platform, passing msg.data into msg hidden div
+app.clientside_callback("function(msg){if(msg == null) {return;} else {return msg.data;}}",
```

Comment:
Following our discussion, I think the reason why they write this callback as a [clientside callback](https://dash.plotly.com/clientside-callbacks) in the example we followed is in case the callback is triggered very often. If I understood correctly, those who wrote the extension are the same who rewrote the Dash source code to use WebSockets instead of HTTP requests to introduce interaction in the app. They were looking for performance gains and this should be allowed by the clientside callback. 

However, I feel like we do not really fall into the use cases described in the Dash documentation as the callback should not be triggered very often. I would thus advocate for transforming it in a plain Python callback. I mean not necessarily in this PR but I just wanted to raise this point and get your opinion on the matter!

Reply:
Why not, I honestly don't have any strong opinion on this but I agree that for code unity it might be better to use 100% Python callbacks.

Ok for changing it in a next PR !